.env

```diff
@@ -14,7 +14,7 @@ REACT_APP_AUTH_COOKIE=user
 REACT_APP_AUTH_COOKIE_EXPIRES=0.5
 REACT_APP_AJAX_TIMEOUT=60000
 REACT_APP_TOAST_NOTIFICATIONS_TIMEOUT=8000
-REACT_APP_POLL_INTERVAL=120000
+REACT_APP_POLL_INTERVAL=60000
```

Comment:
comment/optional: apologies I missed the statement from @mirekdlugosz that the old timer was 1 minute. 

the old timer was this setting `120000` we updated a couple of years back, [see the blame](https://github.com/quipucords/quipucords-ui/blame/1.8.0/.env#L15). Whether it was working correctly that's a parallel issue. But if we're trying to mirror the old behavior we should just leave the value as is... if we want more refreshes I'm not necessarily opposed

Reply:
I believe it was a consensus that 1 minute poll intervals was ideal. I will keep REACT_APP_POLL_INTERVAL=60000